#! /bin/sh /usr/share/dpatch/dpatch-run
## 396180_remove_unused_header.dpatch by Martin Pitt <mpitt@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix possible future FTBFS by removing an unused header.

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0

diff -urNad hfsplus-1.0.4~/libhfsp/src/unicode.c hfsplus-1.0.4/libhfsp/src/unicode.c
--- hfsplus-1.0.4~/libhfsp/src/unicode.c	2002-03-05 20:50:29.000000000 +0100
+++ hfsplus-1.0.4/libhfsp/src/unicode.c	2006-10-30 18:50:04.181626115 +0100
@@ -16,7 +16,6 @@
 #include <stdlib.h>
 #include <endian.h>
 #include <byteswap.h>
-#include <linux/string.h>
 
 #define __USE_GNU
     /* need wcsrtomb */
